### PR TITLE
_ex.yml template created

### PR DESCRIPTION
**Pull request by rubenvecg**
Created a template that can serve as documentation on how to create config files. Any feedback on the content would be appreciated!

### CODE REVIEW

File: library/templates/_ex.yml [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+  # save name value in &name alias
+  name: &name agencyinitials_datasetname  
+  # {{ version }} variable will get its value from either manual input or default formatting 
+  version: "{{ version }}"  
+  # "public-read" for publicly available datasets. Otherwise, it would be "private".
+  acl: public-read | private
+  
+  source:
+    ## Data can come from different types of sources
+    # If data comes from the NYC open data site, then use the socrate field, specifying the uid and file formar
```
Comment (mgraber) [inline library/templates/_ex.yml:14]: socrate -> socrata

File: library/templates/_ex.yml [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+    ## Some datasets are scraped from the web of obtained through a script saved on library/script
+    # such scripts will have the same name formatting as this file and the name field. Therefore,
+    # use a reference of the name alias with &name 
+    script: *name
+  
+    # Geometry data parameters
+    # Examples:
+    # SRS: EPSG:4326
+    # type: MULTIPOLYGON
```
Comment (mgraber) [inline library/templates/_ex.yml:41]: Could include guidance about likely source SRS (opendata is usually 4326, bytes is usually 2263).

File: library/templates/_ex.yml [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+      - "AUTODETECT_TYPE=NO"
+      - "EMPTY_STRING_AS_NULL=YES"
+  
+  destination:
+    # New dataset will keep the same name. Reference the name alias with *
+    name: *name
+
+    # Geometry data for new dataset. might be the same values from above or different.
+    # Examples:
+    # SRS: EPSG:4326
```
Comment (mgraber) [inline library/templates/_ex.yml:58]: I think for our own internal use, default is 4326. Might be worth mentioning.

File: library/templates/_ex.yml [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+    geometry:
+      SRS:
+      type:
+    
+    # Layer creation options. Default values are below.
+    options:
+      - "OVERWRITE=YES"
+      - "PRECISION=NO"
+
+    # If fields will be renamed, they are listed here as a python list. If not, list is empty.
```
Comment (mgraber) [inline library/templates/_ex.yml:69]: I would remove 'python'

File: library/templates/_ex.yml [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+    fields: []
+
+    sql: null
+
+  # Metadata
+  info:  
+    description:
+
+    # If data is dependent on other sources, they are listed as a python list. If not, list is empty.
```
Comment (mgraber) [inline library/templates/_ex.yml:78]: Again, I would remove 'python'

**Comment by SPTKL** [comment]
@rubenvecg  note that everything inside of the `library` folder will end up in the python package, a file like this should be living in a seperate folder under root called `examples`

**Pull request closed without merging**